mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/loanschedule/domain/AbstractLoanScheduleGenerator.java

```diff
@@ -475,7 +475,7 @@ else if (detail.isLatePayment() && detail.isOverlapping(periodStartDate, schedul
             instalmentNumber++;
         }
 
-        if (principalDisbursed.isNotEqualTo(expectedPrincipalDisburse) && diffAmt == null) {
+        if ((!loanApplicationTerms.isMultiDisburseLoan()) && principalDisbursed.isNotEqualTo(expectedPrincipalDisburse) && diffAmt == null) {
```

Comment:
this validation is applicable only for multi disburse loan. if you are blocking that means it's not required. you can remove this block